tests/unittests/distros/test_manage_service.py

```diff
@@ -28,6 +30,14 @@ def test_manage_service_service_initcmd(self, m_subp, m_sysd):
         self.dist.manage_service("start", "myssh")
         m_subp.assert_called_with(["service", "myssh", "start"], capture=True)
 
+    @mock.patch.object(MockDistro, "uses_systemd", return_value=False)
+    @mock.patch("cloudinit.distros.subp.subp")
+    def test_manage_service_rcctl_initcmd(self, m_subp, m_sysd):
+        self.dist = _get_distro("openbsd")
```

Comment:
Nit: since you're not using the one created by setup, we don't really need the self here (though it's not hurting anything).